README.md

```diff
@@ -354,6 +354,76 @@ class HelloWorld {
 }
 ```
 
+### %M for Members
+
+Similar to types, KotlinPoet has a special placeholder for **members** (functions and properties) 
+that are accessible in static context - this includes top-level members and members declared inside
+objects. Use **`%M`** to reference members, pass an instance of `MemberName` as the argument for 
+the placeholder, and KotlinPoet will handle imports automatically:
+
+```kotlin
+val createTaco = MemberName("com.squareup.tacos", "createTaco")
+val isVegan = MemberName("com.squareup.tacos", "isVegan")
+val file = FileSpec.builder("com.squareup.example", "TacoTest")
+    .addFunction(FunSpec.builder("main")
+        .addStatement("val taco = %M()", createTaco)
+        .addStatement("println(taco.%M)", isVegan)
+        .build())
+    .build()
+println(file)
+```
+
+The code above generates the following file:
+
+```kotlin
+package com.squareup.example
+
+import com.squareup.tacos.createTaco
+import com.squareup.tacos.isVegan
+
+fun main() {
+    val taco = createTaco()
+    println(taco.isVegan)
+}
+```
+
+As you can see, it's also possible to use `%M` to reference extension functions and properties. You 
+just need to make sure the member can be imported without simple name collisions, otherwise 
+importing will fail and the code generator output will not pass compilation. There's a way to work 
+around such cases though - use `FileSpec.addAliasedImport()` to create an alias for a clashing 
+`MemberName`:
+
+```kotlin
+val createTaco = MemberName("com.squareup.tacos", "createTaco")
+val isVegan = MemberName("com.squareup.tacos", "isVegan")
+val isVegan2 = MemberName("com.squareup.tacos2", "isVegan")
```

Comment:
would it be a better example if it was `tacos.isVegan` and maybe `cake.isVegan` ? A more plausible conflict?

Reply:
Yeah!